### DISCUSSION TITLE
Possible to use with `viem`?

### DISCUSSION THREAD

**Discussion by o-az** [wevm/wagmi#2159] [Q&A] [answered]
Docs say to install `ethers@^5`. So to be sure, it's not possible to use `wagmi` with `viem` right now?

**Comment by jxom** [comment, marked as answer]
Not yet, but [soon](https://github.com/wagmi-dev/wagmi/pull/2135)!

**Reply by jxom** [reply]
https://github.com/wagmi-dev/wagmi/discussions/2207
